index.html

```diff
@@ -136,7 +136,7 @@ <h3 class="fadeIn animated wow" data-wow-delay=".1s">Designed and built a two-st
 
       <div class="col-sm-6">
         <img src="{{ site.baseurl }}/images/Tanty.jpg" class="img-responsive frame-border" class="img-responsive fadeIn animated wow" data-wow-delay=".2s">
-        <p class="text-muted image-caption">UBC Rocket's first two-stage rocket, launched in 2022 at the first Launch Canada competition. Photo credit from Launch Canada organisers. </p>
+        <p class="text-muted image-caption">UBC Rocket's first two-stage rocket, was launched in 2022 at the first Launch Canada competition. Photo credit from Launch Canada organizers. </p>
```

Comment:
either remove the "was" or remove the redundant comma so that it becomes "rocket was" - my suggestion is leave this as it is